near-sdk/src/store/unordered_map/mod.rs

```diff
@@ -28,7 +28,7 @@ const ERR_NOT_EXIST: &str = "Key does not exist in map";
 /// The default hash function for [`UnorderedMap`] is [`Sha256`] which uses a syscall
 /// (or host function) built into the NEAR runtime to hash the key. To use a custom function,
 /// use [`with_hasher`]. Alternative builtin hash functions can be found at
-/// [`near_sdk::crypto_hash`](crate::crypto_hash).
+/// [`near_sdk::store::key`](crate::store::key).
```

Comment:
By the way, don't forget to update the docs once you are set on namings and which traits are staying (e.g. `CryptoHasher` -> `ToKey`)

Reply:
The docs are generated through the rust doc comments here, so they get updated on release. What here would need to change?